config/initializers/content_security_policy.rb

```diff
@@ -10,6 +10,7 @@
   config.content_security_policy do |policy|
     local_domains = %w[https://publish.localhost https://find.localhost]
     prod_domains = [
+      'https://*.find-teacher-training-courses.service.gov.uk',
```

Comment:
Does this match 'https://find-teacher-training-courses.service.gov.uk/' ?

Reply:
Yes it does from my testing on the review and local app